pkg/apis/servicecatalog/types.go

```diff
@@ -267,27 +261,18 @@ type BindingSpec struct {
 	// InstanceRef is the reference to the Instance this binding is to.
 	// Immutable.
 	InstanceRef kapi.ObjectReference
-	// AppLabelSelector selects the pods in the Binding's namespace that
-	// should be injected with the results of the binding.  Immutable.
-	AppLabelSelector metav1.LabelSelector
```

Comment:
PIP? We're not now, nor (per my current understanding) will we be injecting directly into pods. We inject into a secret instead-- the one named by the `SecretName` field in this same struct. So this `AppLabelSelector` seems to be a relic of when we thought we might inject directly into pods.